libs/template/testdata/urlparse-function/template/hello.tmpl

```diff
@@ -0,0 +1,3 @@
+{{ with url "https://www.databricks.com/a/b?o=123#my-fragment" -}}
+{{- print .Scheme `://` .Host -}}
+{{- end}}
```

Comment:
Or you can use `{{-` and `-}}` everywhere and it won't emit spaces/newlines at all.